### PR TITLE
Default status and fixin update

### PR DESCRIPTION
**Pull request by cesar82**

### CODE REVIEW

**Review by fabian202 — APPROVED** [review]

File: src/components/pages/ProgramDetailsPage/ProgramDetail/index.js [hunk 1]
```diff
@@ -1,45 +1,47 @@
...
 import { Col, Form, Row, Tab } from 'react-bootstrap';
 import { OrgImage, ProgramName, ProgramTabs } from './style';
 import React, { useEffect, useState } from 'react';
-import {useForm} from 'react-hook-form';
-import moment from 'moment';
-import Loader from 'react-loader-spinner';
-import omitEmpty from 'omit-empty';
 import { isEmpty, map, omit } from 'lodash';
+
+import CriteriaEditor from '../CriteriaEditor';
```
Comment (normandanielmq) [inline src/components/pages/ProgramDetailsPage/ProgramDetail/index.js:18]: unused import

File: src/components/pages/ProgramDetailsPage/ProgramDetail/index.js [hunk 2]
```diff
@@ -61,21 +63,12 @@ const ProgramDetail = ({
     validationSchema: ProgramSchema,
     defaultValues: {
       eligibilityCriteria: buildDefaults(),
+      status: statuses.find((c) => (c.value === program?.statusId) || c.label === STATUS_DRAFT)
```
Comment (normandanielmq) [inline src/components/pages/ProgramDetailsPage/ProgramDetail/index.js:66]: Could we isolate this function inside the useProgram hook? something like the buildDefault method used for criteria but for the program.

File: src/components/pages/ProgramDetailsPage/ProgramDetail/index.js [hunk 3]
```diff
@@ -48,23 +47,20 @@ const ProgramDetail = ({
...
   const { register, errors, handleSubmit, setValue, control, watch } = useForm({
     validationSchema: ProgramSchema,
     defaultValues: {
-      eligibilityCriteria: buildDefaults(),
-      status: statuses.find(
-        (c) => c.value === program?.statusId || c.label === STATUS_DRAFT
-      ),
-      enabled: program?.enabled || false,
-      visibility: program?.visibility === CHECK_VISIBLE || false,
+      ...buildScholarshipDefault(statuses, program?.statusId, program?.enabled, program?.visibility),
```
Comment (normandanielmq) [inline src/components/pages/ProgramDetailsPage/ProgramDetail/index.js:62]: Do not send field by field, this is going to be huge, just send the entire program object

Comment (cesar82) [reply src/components/pages/ProgramDetailsPage/ProgramDetail/index.js:62]: 😂😂

**Review by normandanielmq — APPROVED** [review]

**Pull request merged**